firmware/application/apps/ais_app.hpp

```diff
@@ -164,13 +164,11 @@ class AISAppView : public View {
     std::string title() const override { return "AIS RX"; };
 
    private:
-    static constexpr uint32_t initial_target_frequency = 162025000;
```

Comment:
Apps setting an initial frequency should be left doing it.
It's going to make a lot of noise if they can't catch AIS or ADSB which do not allow th change the frequency in app

Reply:
I'll try to come up with a better pattern. For now, this will do:
```
    if (!settings_.loaded())
        receiver_model.set_target_frequency(initial_target_frequency);
```